packages/atoms/src/ListItem/listItemTypes.ts

```diff
@@ -0,0 +1,11 @@
+import { IconType } from '../Icon/iconTypes';
+import { NativeStyleObject } from '@trezor/styles';
+
+export type BaseListItem = {
+    iconType?: IconType;
+    title: string;
+    subtitle?: string;
+    onPress?: () => void;
+    style?: NativeStyleObject;
+    isTextWrapped?: boolean;
```

Comment:
I have problem to tell from name what is this doing. Text is wrapped always is something :D 

I suggest name like `isTextTruncated`